search-enterprise/latest/config/language.md

```diff
@@ -58,37 +58,44 @@ To create a custom configuration you must create a `JSON` file in the locale con
     "analysis": {
       "analyzer": {
         "locale_content": {
-          "type": "french"        },
+          "type": "french"
+        },
         "locale_text_index": {
           "tokenizer": "whitespace",
           "filter": [
             "asciifolding",
             "custom_word_delimiter_graph",
             "lowercase",
             "flatten_graph",
-            "french_stemmer"          ]
+            "french_stemmer_multiplexer"
+          ]
         },
         "locale_text_query": {
           "tokenizer": "whitespace",
           "filter": [
             "asciifolding",
             "custom_word_delimiter_graph",
             "lowercase",
-            "french_stemmer"          ]
+            "french_stemmer_multiplexer"
+          ]
         }
       },
       "filter": {
         "french_stemmer": {
           "type": "stemmer",
-          "language": "light_french"        }
+          "language": "light_french"
+        },
+        "french_stemmer_multiplexer": {
+          "type": "multiplexer",
+          "filters": [ "french_stemmer" ]
+        }
       }
     }
   }
 }
 ```
 
-The `analysis` section contains three analyzers and any custom defined filters. Each analyzer section must contain at least one tokenizer. You can optionally include a number of filters which can then modify the tokens, for example to convert text to lowercase for the index. `locale_content` is a symmetric content analyzer, which in this example is set to `french`. `locale_text_index` is an asymmetric text analyzer, which is set to use whitespace as the delimiter, and has several filters including a custom defined filter, in this case called `french_stemmer`.
-The `locale_text_query` section contains an asymmetric text query analyzer which is also set to use whitespace as the delimiter. It has several filters, including, in this case, one called `french_stemmer`.
+The `analysis` section contains three analyzers and any custom defined filters. Each analyzer section must contain at least one tokenizer. You can optionally include a number of filters which can then modify the tokens, for example to convert text to lowercase for the index. `locale_content` is a symmetric content analyzer, which in this example is set to `french`. `locale_text_index` is an asymmetric text analyzer, which you set to use whitespace as the delimiter. It has several filters including a custom defined filter, in this instance its called `french_stemmer_multiplexer`. The `locale_text_query` section contains an asymmetric text query analyzer which is also set to use whitespace as the delimiter. It has several filters, including, in this case, one called `french_stemmer_multiplexer`. The filter section contains any custom defined filters that are used by the analyzers. In this case the filter is called `french_stemmer_multiplexer`.
 The `filter` section contains any custom defined filters that are to be used by the analyzers. In this case the filter is called `french_stemmer`.
```

Comment:
This line should be removed because it's duplicated (we should mention about `french_stemmer_multiplexer` instead of `french_stemmer`).

Reply:
hehe thought so removed ! thanks